packages/x-components/src/components/result/base-result-rating.vue

```diff
@@ -2,6 +2,7 @@
   <component
     :is="link ? 'a' : 'div'"
     v-if="result.rating && result.rating.value"
+    ref="root"
```

Comment:
For consistency, keep the same name for the refs of the root element. Here we use `root` and in other component `el`, should be the same in both (I prefer `el` but it's your call).